sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/CommunityGalleryTests.cs

```diff
@@ -99,6 +99,7 @@ private void ValidateCommunityGallery(CommunityGallery communityGallery)
         {
             string expectedId = "/CommunityGalleries/" + PublicGalleryName;
             Assert.Equal(expectedId, communityGallery.UniqueId);
+            Assert.NotNull(communityGallery.Disclaimer);
```

Comment:
Can you have this test check if the Disclaimer value is a specific value rather than just not null?
If the Disclaimer value changes often given the same setup, maybe this not null is fine, but I guess this is not the case.

Reply:
added